packages/core/src/error.ts

```diff
@@ -1,26 +1,53 @@
+import { PROBLEM_DETAILS } from "./problem-details-registry"
+
+/**
+ * Represents a problem details type.
+ *
+ * This implementation does not yet support:
+ * - instance member
+ * - extension members
+ *
+ * @see https://datatracker.ietf.org/doc/html/rfc9457#name-the-problem-details-json-ob
+ */
+export interface ProblemDetailsType {
```

Comment:
I am not sure if this belongs here or in `packages/core/src/problem-details-registry.ts`

Reply:
```suggestion
export type ProblemDetail = Pick<ApplicatioNError, 'type' | 'title' | 'status'>
```

If something is called `WhateverType` then the Type part can probably be removed.